cmp/filter/impl/Option.ts

```diff
@@ -17,8 +34,8 @@ import {FieldType} from '../../../data';
  * Create an option representing a detailed field suggestion
  * @returns {FilterChooserOption}
```

Comment:
all these returns are redundant now, have been removing if only thing it specifies is type